app/views/locations/new.erb

```diff
@@ -18,7 +18,14 @@
   <div class='govuk-grid-column-two-thirds'>
     <%= link_to 'Back to list', ips_path, class: 'govuk-back-link' %>
 
-    <h2 class='govuk-heading-l'>Add a new location</h2>
+    <% if current_organisation.locations.empty? %>
+      <h2 class='govuk-heading-l'>Add a new location</h2>
+      <div class="govuk-inset-text">
```

Comment:
I'm not sure if the `inset-text` styling is useful to highlight this message or not. I'd try it as just a paragraph and see what it looks like.

Reply:
In case confusing in lieu of later comments - this is only a relevant comment if you don't agree with my recommendation to remove this text!